From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: ColorfulRhino <131405023+ColorfulRhino@users.noreply.github.com>
Date: Sat, 30 Mar 2024 21:09:12 +0100
Subject: [ARCHEOLOGY] meson-s4t7: Fix custom_kernel_config: hash modification
 has to happen inside the first function call

> X-Git-Archeology: > recovered message: > Will be ignored on second time the function is called
> X-Git-Archeology: - Revision 6ced5cc02637fb6dbc980aa77fcdc77f2ccf5067: https://github.com/armbian/build/commit/6ced5cc02637fb6dbc980aa77fcdc77f2ccf5067
> X-Git-Archeology:   Date: Sat, 30 Mar 2024 21:09:12 +0100
> X-Git-Archeology:   From: ColorfulRhino <131405023+ColorfulRhino@users.noreply.github.com>
> X-Git-Archeology:   Subject: meson-s4t7: Fix custom_kernel_config: hash modification has to happen inside the first function call
> X-Git-Archeology:
> X-Git-Archeology: - Revision caf0529240948df416b015aeea8c23e420a55ce6: https://github.com/armbian/build/commit/caf0529240948df416b015aeea8c23e420a55ce6
> X-Git-Archeology:   Date: Sun, 31 Mar 2024 18:21:26 -0400
> X-Git-Archeology:   From: Barry Lind (SteeManMI) <barrylind@yahoo.com>
> X-Git-Archeology:   Subject: Bump meson64 edge from 6.7 to 6.8
> X-Git-Archeology:
---
 tools/cgroup/Makefile | 11 ++++++++++
 1 file changed, 11 insertions(+)

diff --git a/tools/cgroup/Makefile b/tools/cgroup/Makefile
new file mode 100644
index 000000000000..111111111111
--- /dev/null
+++ b/tools/cgroup/Makefile
@@ -0,0 +1,11 @@
+# SPDX-License-Identifier: GPL-2.0
+# Makefile for cgroup tools
+
+CFLAGS = -Wall -Wextra
+
+all: cgroup_event_listener
+%: %.c
+	$(CC) $(CFLAGS) -o $@ $^
+
+clean:
+	$(RM) cgroup_event_listener
-- 
Armbian

